core/src/stores/store.rs

```diff
@@ -669,6 +669,12 @@ pub const SQL_INDEXES: [&'static str; 30] = [
         idx_data_sources_nodes_data_source_node_id ON data_sources_nodes(data_source, node_id);",
     "CREATE INDEX IF NOT EXISTS
         idx_data_sources_nodes_parents_array ON data_sources_nodes USING GIN (parents);",
+    "CREATE INDEX CONCURRENTLY IF NOT EXISTS
```

Comment:
these are now only used at table init. Concurrently not really needed but if we want to add it let's add it everywhere out of consistency

Reply:
removing them